.github/workflows/windows.yml

```diff
@@ -20,10 +20,17 @@ jobs:
         qt: [ qt5, qt6 ]
     runs-on: windows-latest
     name: windows-latest ${{ matrix.qt }}
+    env:
+      MSYS_DIR: 'C:/openscad'
```

Comment:
Do we know that this is always a good path?

The msys2 action has a way of querying this: https://github.com/msys2/setup-msys2?tab=readme-ov-file#msys2-location